governance/polls/MIP9 Community Greenlight Poll - OHM (Olympus DAO) - November 1, 2021.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Community Greenlight Poll - OHM (Olympus DAO) - November 1, 2021
+summary: Signal your support or opposition to prioritising onboarding $asset_code ($asset_name). 
+discussion_link: https://forum.makerdao.com/t/ohm-olympus-dao-collateral-onboarding-application-mip6/10709
+vote_type: Plurality Voting
+categories:
+   - Collateral
+   - Greenlight
+options:
+   0: Abstain
+   1: Yes (Greenlight)
+   2: No (Defer)
+start_date: 2021-11-01T16:00:00
+end_date: 2021-11-15T16:00:00
+---
+# Poll: Community Greenlight Poll - OHM (Olympus DAO) - November 1, 2021
+
+The Governance Facilitators have placed a governance poll into the [voting system](https://vote.makerdao.com/polling) as part of their responsibilities as defined in [MIP9: Community Greenlight](https://mips.makerdao.com/mips/details/MIP9). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, November 1 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+This poll asks whether the asset OHM (Olympus DAO) - November 1, 2021 should be prioritized for inclusion in the Maker Protocol by the [Core Units](https://mips.makerdao.com/mips/details/MIP38#mip38c2-core-unit-state). 
```

Comment:
Don't think we need the date in this line

Reply:
Hey Mkr, im out right now. Ill males the changes in a couple or hours. If thats too late feel free to do them yourself

Reply:
I’ll be doing the final review submission in ~8 hours so no rush! Just let me know if you don’t have a chance to update

Reply:
Don't think I have write access to your repo